python/requirements_compiled.txt

```diff
@@ -471,12 +471,6 @@ fairscale==0.4.6
     # via -r /ray/ci/../python/requirements/ml/tune-test-requirements.txt
 farama-notifications==0.0.4
     # via gymnasium
-fastapi==0.104.0
-    # via
-    #   -r /ray/ci/../python/requirements.txt
-    #   -r /ray/ci/../python/requirements/test-requirements.txt
-    #   aim
-    #   gradio
```

Comment:
are the packages entirely removed from the dependencies?

Reply:
IIUC, this is the constraints file, if remove here, we will use latest version of fastapi. https://github.com/ray-project/ray/blob/master/python/setup.py#L269

Reply:
talk to @can-anyscale (or me) if you need help on how to generate / recompile this file.

Reply:
you can update the content using this file https://buildkite.com/ray-project/premerge/builds/16643#018d149d-9eb8-4edf-a1a7-29c2e46a25dd